psalm.xml

```diff
@@ -2,7 +2,6 @@
 <psalm
     totallyTyped="true"
     errorLevel="5"
-    resolveFromConfigFile="true"
```

Comment:
Is this default in newer versions? Otherwise I think we should keep it, since the working dir is probably not always clear.

Reply:
It is